### PR TITLE
Added information to pathogenesis

### PR DESCRIPTION
**Pull request by davemai**
Added some details to viral entry and spreading

### Suggested reviewers (optional)

@rando2 

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by rando2** [comment]
@anskelly @jinhui2 would either/both of you be able to review this addition to the section you wrote?

**Comment by jinhui2** [comment]
The sentence in the "viral entry" regarding the ACE2 expression in different organs is a great addition. 
The paragraphs @davemai added in the section "Viral Replication and Spreading" indicate the virus spreading can be potentially enhanced by ACE2 up-regulation or inflammatory cytokines stimulation. It's part of the story for this section. Before these two paragraphs, we need to add information/mechanism of how the virus replicate within the cell and spread to neighbor cells. After these paragraphs, we may also need to add some information about the person to person transmission routes.

**Comment by rando2** [comment]
> The sentence in the "viral entry" regarding the ACE2 expression in different organs is a great addition.
> The paragraphs @davemai added in the section "Viral Replication and Spreading" indicate the virus spreading can be potentially enhanced by ACE2 up-regulation or inflammatory cytokines stimulation. It's part of the story for this section. Before these two paragraphs, we need to add information/mechanism of how the virus replicate within the cell and spread to neighbor cells. After these paragraphs, we may also need to add some information about the person to person transmission routes. What do you guys think?

@jinhui2 that sounds great to me. I really appreciate that you're thinking about the larger narrative! Would you want to file a PR suggesting some changes to the outline (e.g., add in new sections by adding new headers following the `###`, `####`, etc. format) or add in questions that need to be answered below the respective headers?

File: content/07.pathogenesis.md [hunk 1]
```diff
@@ -21,13 +22,19 @@ The C-terminal domain of the SARS-CoV-2 S protein in particular was identified a
...
 Recent crystal structures have shown that the receptor-binding domain (RBD) of SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507]. 
 Because the RBD plays such a critical role in viral entry, blocking its interaction with ACE2 represents a promising therapeutic approach. Nevertheless, despite the high structural homology between SARS-CoV-2 RBD and that from SARS-CoV, monoclonal antibodies targeting SARS-CoV-RBD failed to bind SARS-CoV-2-RBD [@doi:10.1126/science.abb2507]. 
-Promisingly though, sera from convalescent SARS patients inhibited SARS-CoV-2 viral entry in vitro, albeit with lower efficiency than it inhibited SARS-CoV [@doi:10.1016/j.cell.2020.02.052]. 
+Promisingly though, sera from convalescent SARS patients inhibited SARS-CoV-2 viral entry in vitro, albeit with lower efficiency than it inhibited SARS-CoV [@doi:10.1016/j.cell.2020.02.052].
 
 #### Viral Replication and Spreading
```
Comment (jinhui2) [inline content/07.pathogenesis.md:28]:
```suggestion
##### Viral replication cycle within cells

##### Viral spreading among cells
```

File: content/07.pathogenesis.md [hunk 2]
```diff
@@ -21,13 +22,19 @@ The C-terminal domain of the SARS-CoV-2 S protein in particular was identified a
...
 
 -Basic introduction into replication cycle
 -What are the routes of transmission
```
Comment (jinhui2) [inline content/07.pathogenesis.md:31]:
```suggestion
##### Mechanism of viral replication within cells
##### Mechanism of viral spreading to neighbor cells
##### Factors enhancing viral spreading
```

File: content/07.pathogenesis.md [hunk 3]
```diff
@@ -21,13 +22,19 @@ The C-terminal domain of the SARS-CoV-2 S protein in particular was identified a
...
+In several preliminary assays and an analysis of previous microarray data, Wang et. al. reported that ACE2 is significantly upregulated following infection of other coronaviruses, including SARS-CoV and MERS-CoV, as well as viruses such as rhinovirus and influenza virus [@doi:10.1101/2020.02.24.963348].
+Additionally, direct stimulation with inflammatory cytokines such as type I interferons resulted in upregulation of ACE2, with treated groups showing 4-fold higher ACE2 expression as compared to control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].
+Though whether SARS-CoV-2 infection facilitates positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies. 
+One severe example includes reports of cytokine storm-like responses in patients with particularly severe infections, in which the overproduction of inflammatory cytokines leads to systemic inflammation and potentially multi-organ failure, and may very well accelerate the spread of virus in the host [@doi:10.1101/2020.02.24.963348; @doi:10.1016/S0140-6736(20)30628-0].
+
```
Comment (jinhui2) [inline content/07.pathogenesis.md:37]:
```suggestion
##### Person-to-person transmission
```

**Comment by davemai** [comment]
@jinhui2 yes, I agree the context you suggest is needed - I came across some information during my reading and thought it would be good to add, but haven't read anything about SARS-CoV-2 specific mechanisms of viral replication yet. The subtitles are good to key people in to information they might have come across that is relevant!

File: content/07.pathogenesis.md [hunk 4]
```diff
@@ -21,13 +22,21 @@ The C-terminal domain of the SARS-CoV-2 S protein in particular was identified a
...
+In several preliminary assays and an analysis of previous microarray data, Wang et. al. reported that ACE2 is significantly upregulated following infection of other coronaviruses, including SARS-CoV and MERS-CoV, as well as viruses such as rhinovirus and influenza virus [@doi:10.1101/2020.02.24.963348].
+Additionally, direct stimulation with inflammatory cytokines such as type I interferons resulted in upregulation of ACE2, with treated groups showing 4-fold higher ACE2 expression as compared to control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].
+Though whether SARS-CoV-2 infection facilitates positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies. 
+One severe example includes reports of cytokine storm-like responses in patients with particularly severe infections, in which the overproduction of inflammatory cytokines leads to systemic inflammation and potentially multi-organ failure, and may very well accelerate the spread of virus in the host [@doi:10.1101/2020.02.24.963348; @doi:10.1016/S0140-6736(20)30628-0].
```
Comment (rando2) [inline content/07.pathogenesis.md:38]:
```suggestion
One severe example includes reports of cytokine storm-like responses in patients with particularly severe infections, in which the overproduction of inflammatory cytokines leads to systemic inflammation and potentially multi-organ failure, and may very well accelerate the spread of virus in the host [@doi:10.1101/2020.02.24.963348; @doi:10/ggnzmc].
```

**Comment by cgreene** [comment]
I just made some quick changes to fix the list and header formatting issues I was seeing. Let's see what it looks like once it builds.

Screen Shot 2020-04-20 at 7 48 05 AM

**Comment by cgreene** [comment]
Success!

Screen Shot 2020-04-20 at 7 56 14 AM

File: content/07.pathogenesis.md [hunk 5]
```diff
@@ -21,12 +22,26 @@ The C-terminal domain of the SARS-CoV-2 S protein in particular was identified a
...
+In several preliminary assays and an analysis of previous microarray data, Wang et. al. reported that ACE2 is significantly upregulated following infection of other coronaviruses, including SARS-CoV and MERS-CoV, as well as viruses such as rhinovirus and influenza virus [@doi:10.1101/2020.02.24.963348].
+Additionally, direct stimulation with inflammatory cytokines such as type I interferons resulted in upregulation of ACE2, with treated groups showing 4-fold higher ACE2 expression as compared to control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].
+Though whether SARS-CoV-2 infection facilitates positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies. 
+One severe example includes reports of cytokine storm-like responses in patients with particularly severe infections, in which the overproduction of inflammatory cytokines leads to systemic inflammation and potentially multi-organ failure, and may very well accelerate the spread of virus in the host [@doi:10.1101/2020.02.24.963348; @doi:10/ggnzmc].
```
Comment (cgreene) [inline content/07.pathogenesis.md:42]: We also have some elements on the cytokine storm in the Tocilizumab section: https://greenelab.github.io/covid19-review/#tocilizumab

I have created an issue to track a potential solution (filling in 2.2): https://github.com/greenelab/covid19-review/issues/233

**Comment by cgreene** [comment]
@davemai : want to take a look at https://github.com/greenelab/covid19-review/pull/224#discussion_r411318342 and see if you agree? If so, I think we could merge after that commit goes in.

**Comment by davemai** [comment]
> @davemai : want to take a look at [#224 (comment)](https://github.com/greenelab/covid19-review/pull/224#discussion_r411318342) and see if you agree? If so, I think we could merge after that commit goes in.

**Comment by davemai** [comment]
@cgreene the edit is good! I accidentally closed this pull request and reopened it as I was trying to respond to your comment ...

**Pull request merged**